{"event": "verbose", "uuid": "40628abd-d4f4-485c-8c1a-ae678bf9aebe", "counter": 6, "stdout": "\u001b[0;31mThe error appears to be in '/home/duchess/Documents/src/github.com/edwardtheharris/ansible-kcp/tmp_dir_sphinx_ansible/temp_file.yaml': line 1, column 3, but may\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "a0f21c06-f617-458b-b9aa-0e7b67c011ee", "created": "2024-07-23T18:49:24.402755+00:00"}